### DISCUSSION TITLE
I am applying for student developer pack but it says use your device camera but the rear facing camera option doesn't work

### DISCUSSION THREAD

**Discussion by thedhamibro** [community/community#54156] [GitHub Education]
### Hi everyone,

I am applying for student development pack but it gives the error saying use your device camera and when I try to use it , it just opens the selfie camera and the option to change the camera to rear camera doesn't work,what should I do? And it is not my mobiles problem, I tried it on my mother's mobile and it didn't work their either.

### A suggestion...(optional)

Use opera browser in mobile, it works there

**Comment by patgd** [comment]
I also have this issue.

**Comment by Gada25** [comment]
Gada

**Comment by Gada25** [comment]
Gada[

**Reply by Gada25** [reply]
Y

**Comment by Gada25** [comment]
Gada

**Comment by advayc** [comment]
If you have a mobile device take a picture of your id then convert it into a jpg file and upload. Take the phot with your camera app and upload it there instead of taking the picture with the github popup.

**Reply by Gada25** [reply]
Ifa

**Reply by patgd** [reply]
Unfortunately, there is no option to upload an image on the desktop or mobile site. Additionally, the option to select the rear camera on the mobile site fails to access the phone's rear camera.

**Reply by ghost** [reply]
yes

**Reply by thedhamibro** [reply]
did that, but got a email saying that use your mobile camera for applying to the pack.

**Comment by ghost** [comment]
I also have this problem, it seems that this problem has been around for a long time, I think this is a bug in github, but I don't know how to report the problem to the official, so that they can fix this problem as soon as possible https://github.com/orgs/community/discussions/64847

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)

**Reply by thedhamibro** [reply]
Tnx❤️ It works
